### DISCUSSION TITLE
Best practices of PHP and MySQL

### DISCUSSION THREAD

**Discussion by ghost** [community/community#108033] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

I am building a StackoverFlow remake and I was going to use PHP, JavaScript, HTML, CSS and MySQL.

And I have an extremely low intelligence

**Comment by gil-son** [comment]
You have to be patient and respect yourself

Do you have any doubt?

**Reply by ghost** [reply]
Kind of because when I try to learn something it takes forever to learn.

**Reply by gil-son** [reply]
It's normal, use some techniques, like:

- reflect about the information
- look by differences ways, like videos, documentations, groups, etc
- if necessary return some steps
- try to create analogies
- if necessary to do diagrams, then build one
- switch information with friends, colleagues or forums
- the practice is very important
- try to explain to yourself or other people

Look that image:

<div align="center">
  <img src="https://www.educationcorner.com/wp-content/uploads/learning-pyramid2.jpg">
</div>

**Reply by ghost** [reply]
### How can I project my MySQL database?

**Reply by gil-son** [reply, marked as answer]
- You need to install SQL in your computer
- Use a IDE to manage the database, for example: workbench, DBeaver, Microsoft SQL, Adminer, etc
- Create a database
- Create a project using a program language
- Connect your project with your data base

I visited your repositories and you are able to do that

If I supported you, consider to mark my answer. Good Luck!

**Comment by ucodefusion** [comment]
Do you have any php framework skill

**Reply by ghost** [reply]
No, I don't know any PHP libraries.

**Reply by ucodefusion** [reply]
It's allwrite , I think if you have time it's good to get idea about laravel, Symfony or codeigniter .
With a php framework your life will be easier and you can do a standard application.

**Comment by senja060995** [comment]
hello

**Reply by ghost** [reply]
Hi
